### PR TITLE
gnome-tweaks: fix libhandy dependency

### PR DESCRIPTION
**Pull request by Roger-Roger-debug**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

The newest version requires libhandy >= 1.0

### CODE REVIEW

File: srcpkgs/gnome-tweaks/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'gnome-tweaks'
 pkgname=gnome-tweaks
 version=40.0
-revision=2
+revision=3
 build_style=meson
 hostmakedepends="gettext"
-depends="gtk+3 dconf gnome-settings-daemon mutter libnotify python3-gobject libhandy"
+depends="gtk+3 dconf gnome-settings-daemon mutter libnotify python3-gobject libhandy1"
 short_desc="GNOME3 tool to customize advanced options"
```
Comment (paper42) [inline srcpkgs/gnome-tweaks/template:8]: Let's also fix this:
```suggestion
 short_desc="GNOME tool to customize advanced options"
```

**Comment by paper42** [comment]
Could you add the email used in the commit to your github profile so gh can associate the commit with you?

**Comment by Roger-Roger-debug** [comment]
> Could you add the email used in the commit to your github profile so gh can associate the commit with you?

Is this okay or does it have to be displayed as the public email?

**Comment by paper42** [comment]
> > Could you add the email used in the commit to your github profile so gh can associate the commit with you?
> 
> Is this okay or does it have to be displayed as the public email?

It's fine like this, could you also please squash the commits into one?

**Comment by paper42** [comment]
thank you for catching this

**Pull request merged**